data/Tars.txt

```diff
@@ -0,0 +1,13 @@
+T 0 return book
+T 0 buy book
+E 0 hi (from: 2019-10-15T14:30 to: 2019-10-16T15:30 )
```

Comment:
not a coding standard issue, but you could consider formatting the dates and times in a more friendly readable format